README.md

```diff
@@ -257,7 +257,7 @@ fixtures:
       ref: "2.6.0"
 ```
 
-Only install the `yumrepo_core` module when testing against Puppet 6 or greater:
```

Comment:
Is this really necessary now that everything below Puppet 7 is unsupported?

Reply:
i originally removed all of this but thought it still had a place since it still seems relevant to >6. Seems now that this module should always be installed since all supported puppets are >6